stack/schema.go

```diff
@@ -22,6 +22,9 @@ type Function struct {
 	// Image Docker image name
 	Image string `yaml:"image"`
 
+	// Docker registry Authorization
+	RegistryAuth string `yaml:"registry_auth,omitempty"`
```

Comment:
I don't think that we need to store credentials in the function like this, for security reason. We might implement a method on gateway so that the gateway can register credentials to docker on the gateway.

Reply:
We are not storing the credentials here, this is just an interface to gateway. The only security problem here would be if the gateway is not using TLS and the `registry_auth` is sent in the plain text. This is why we added a warning about it in order to advise the user to use https instead. The credentials are stored by `docker-cli` and we are just retrieving it... Let me know if you need more info...

Reply:
The credentials is stored in stack.yml, isn't it?

Reply:
@itscaro the credentials are not stored anywhere by us, these credentials are fetch from Docker local storage and send over to the gateway.

Reply:
I don't this is representing the stack.yaml, but the API interface to the gateway. That being said things might have changed since I last looked it in.